test/jackdaw/streams_test.clj

```diff
@@ -603,7 +604,45 @@
             (is (= 3 (count keyvals)))
             (is (= [1 1] (first keyvals)))
             (is (= [1 3] (second keyvals)))
-            (is (= [1 6] (nth keyvals 2)))))))))
+            (is (= [1 6] (nth keyvals 2))))))))
+
+  (testing "suppress records"
```

Comment:
Shall we add another example that covers the use of a suppress config (e.g. max-records, until-time-limit etc).

Reply:
Will do. I'll try to get to it by next Monday!